Makefile

```diff
@@ -107,7 +107,7 @@ $(GOPATH)/bin/gh-release:
 	go get -u github.com/progrium/gh-release/...
 
 $(GOPATH)/bin/go-bindata:
-	go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
It's back now https://github.com/jteeuwen/go-bindata

Reply:
No it's not the original author https://www.reddit.com/r/golang/comments/7vv9zz/popular_lib_gobindata_removed_from_github_or_why/?st=jdd19k82&sh=86e436b0 check the thread.

Reply:
@LalatenduMohanty  Also instead of we fork it into our namespace I would instead use one of maintained fork https://github.com/jteeuwen/go-bindata/issues/3#issuecomment-363927455

Reply:
Use our own fork of go-bindata is the best way forward... I have been paying attention the last few days and that someone was able to recreate the repo and the username in such a short timeframe sounds scary to me. At least the new repo is kinda honest about this: https://github.com/jteeuwen/go-bindata/issues/5 it is a potential backdoor... GitHub should have a read-only archival function., but we should also reconsider our vendor usage.

Reply:
Yes, if the github repo is switched to read-only archival then i don't have anything against using the existing repo, but we should atleast reconsider using our own fork, which is i think is the safest option we have.